darts/metrics/__init__.py

```diff
@@ -19,6 +19,7 @@
         - :func:`MSSE <darts.metrics.metrics.msse>`: Mean Squared Scaled Error
         - :func:`RMSSE <darts.metrics.metrics.rmsse>`: Root Mean Squared Scaled Error
         - :func:`MAPE <darts.metrics.metrics.mape>`: Mean Absolute Percentage Error
+        - :func:`WMAPE <darts.metrics.metrics.wmape>`: Weighted Mean Absolute Percentage Error
```

Comment:
Just for consistency with sMAPE:

```suggestion
        - :func:`wMAPE <darts.metrics.metrics.wmape>`: weighted Mean Absolute Percentage Error
```